sdk/opendp/whitenoise/client/restclient/code-model-v1

```diff
@@ -0,0 +1,1735 @@
+{
```

Comment:
Seems like a new autorest file. I do not think we need it. We should figure out why it exists though. I saw it last time too

Reply:
removed, lol